internal/cmd/login.go

```diff
@@ -78,6 +83,10 @@ $ infra login`,
 				options.Server = args[0]
 			}
 
+			if options.Provider == "infra" {
```

Comment:
The infra provider doesn't seem to work for me. Me guess is setting this blank doesnt see that the provider is set

Reply:
This is the local login. The idea here is if someone has an `INFRA_PROVIDER` defined, if they want to log in to a local user then they specify `--provider infra`. Maybe `"infra"` shouldn't be a valid provider name?

Reply:
Infra as a provider name for this functionality seems good to me. Im just having issues with getting this behavior to work, I may be doing something wrong.

It still prompts me for the provider, but I expect it should just automatically prompt me for username/password:
```
$ export INFRA_PROVIDER=infra 

$ infra login localhost                 
? Select a login method:  [Use arrows to move, type to filter]
> google (accounts.google.com)
  okta (dev.okta.com)
  Login with username and password

 $ infra login localhost --provider infra
? Select a login method:  [Use arrows to move, type to filter]
> google (accounts.google.com)
  okta (dev-50966300.okta.com)
  Login with username and password
```

Reply:
@BruceMacD That worked locally because my local infra instance only had local login. This should be fixed now.